multimodal/src/autogluon/multimodal/data/trivial_augmenter.py

```diff
@@ -278,6 +276,10 @@ def augment_image(self, data):
         return op.augment(scale, data)
 
     def augment_text(self, data):
+        import nlpaug.augmenter.word as naw
+
+        from ..utils.nlpaug import InsertPunctuation
```

Comment:
minor optimization, can have these imported after

```python3
if op == "identity":
       return data
```

That way if we aren't using anything but identity, we don't need to import.

Reply:
Sounds good. Fixed.